src/react/src/util/renderUtils.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const blockStyle = Object.freeze({
+    margin: 0,
+    display: 'block',
+});
+
+const renderUniqueListItems = (value, fieldName = '') => {
+    if (!Array.isArray(value)) {
+        return value;
+    }
+
+    const uniqueValues = [...new Set(value)];
+
+    return uniqueValues.map(v =>
```

Comment:
It is not recommended to use a single letter as a variable name. Could you please provide a more meaningful name instead of `v`?

Reply:
Done.